IBAnimatableTests/FillDesignableTests.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  FillDesignableTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/25/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+final class FillDesignableTests: XCTestCase {
+
+  override func setUp() {
+    super.setUp()
+  }
+
+  override func tearDown() {
+    super.tearDown()
+  }
+
+  // MARK: - testFillColor
+
+  func testFillColor() {
+    helper_testFillColor_UIView(element: AnimatableView())
+    helper_testFillColor_UIView(element: AnimatableScrollView())
+    helper_testFillColor_UIView(element: AnimatableButton())
+    helper_testFillColor_UIView(element: AnimatableCheckBox())
+    helper_testFillColor_UIView(element: AnimatableImageView())
+    //helper_testFillColor_UIView(element: AnimatableStackView()) // broken see #461
```

Comment:
May make some comment like To behavior as same as UIKit, `UIStackView` doesn't respect to `fillColor`.